string-manipulation-2/src/capitalize-word.js

```diff
@@ -0,0 +1,15 @@
+/* exported capitalizeWord */
+// make storage for the capitalized word
+// uppercase the first letter, add the rest of the letters but lowercased and store.
+// if the 4th letter of the word is 's',
+//   make the word JavaScript
+// return location of storage for capitalized word
+
+var capitalWord = '';
+function capitalizeWord(word) {
+  capitalWord = word.slice(0, 1).toUpperCase() + word.slice(1, word.length).toLowerCase();
+  if (capitalWord[4] === 's') {
```

Comment:
This will do this to every word with `s` as its fifth letter, which is **_way_** more words than just JavaScript. You should check for the full word that is the special case, not just one letter of it.